faiss/Index.h

```diff
@@ -20,6 +20,17 @@
 #define FAISS_VERSION_MINOR 8
 #define FAISS_VERSION_PATCH 0
 
+// Macro to combine the version components into a single string
+#ifndef STRINGIFY
+#define STRINGIFY(ARG) #ARG
+#endif
+#ifndef TOSTRING
+#define TOSTRING(ARG) STRINGIFY(ARG)
+#endif
+#define VERSION_STRING            \
```

Comment:
This definition should probably be prefixed with `FAISS_`, otherwise it might collide with other symbols.